src/netex-convertor/netexGenerator.ts

```diff
@@ -117,49 +117,47 @@ const netexGenerator = (
+        // below 2 branding attributes, name and url, are new to point to point, but should not break anything
+        resourceFrameToUpdate.typesOfValue.ValueSet[0].values.Branding.Name.$t = coreData.operatorName;
         resourceFrameToUpdate.typesOfValue.ValueSet[0].values.Branding.Url.$t = coreData.website;
 
-        if (
-            ticket.type === 'multiOperator' &&
-            (isMultiOperatorGeoZoneTicket(ticket) || isSchemeOperatorTicket(ticket))
-        ) {
+        if (coreData.isMultiOperator && (isMultiOperatorGeoZoneTicket(ticket) || isSchemeOperatorTicket(ticket))) {
             const nocs = [...ticket.additionalNocs];
-            if (isMultiOperatorGeoZoneTicket(ticket)) {
-                nocs.push(ticket.nocCode);
+            if (coreData.isGeoZone) {
+                nocs.push((ticket as MultiOperatorGeoZoneTicket).nocCode);
                 resourceFrameToUpdate.organisations.Operator = getOrganisations(operatorData);
-            } else if (isBaseSchemeOperatorInfo(baseOperatorInfo) && isSchemeOperatorTicket(ticket)) {
+            } else if (isBaseSchemeOperatorInfo(baseOperatorInfo) && coreData.isSchemeOperator) {
                 resourceFrameToUpdate.organisations.Operator = getOrganisations(operatorData, baseOperatorInfo);
             }
             resourceFrameToUpdate.groupsOfOperators = getGroupOfOperators(operatorData);
-        } else if (ticket.type === 'multiOperator' && isMultiOperatorMultipleServicesTicket(ticket)) {
-            const nocs: string[] = ticket.additionalOperators.map(additionalOperator => additionalOperator.nocCode);
+        } else if (coreData.isMultiOperator && coreData.isMultiOperatorMultiServices) {
+            const nocs: string[] = ticket.additionalOperators.map(
+                (additionalOperator: { nocCode: any }) => additionalOperator.nocCode,
+            );
             nocs.push(ticket.nocCode);
             resourceFrameToUpdate.organisations.Operator = getOrganisations(operatorData);
             resourceFrameToUpdate.groupsOfOperators = getGroupOfOperators(operatorData);
-        } else if (
-            !isMultiOperatorGeoZoneTicket(ticket) &&
-            !isSchemeOperatorTicket(ticket) &&
-            !isMultiOperatorMultipleServicesTicket(ticket)
-        ) {
+        } else if (!coreData.isMultiOperator && !coreData.isSchemeOperator && !coreData.isMultiOperatorMultiServices) {
             resourceFrameToUpdate.organisations.Operator.id = coreData.nocCodeFormat;
             resourceFrameToUpdate.organisations.Operator.PublicCode.$t = coreData.operatorIdentifier;
-            resourceFrameToUpdate.organisations.Operator.Name.$t = operatorPublicName;
-            resourceFrameToUpdate.organisations.Operator.ShortName.$t = ticket.operatorName;
+            resourceFrameToUpdate.organisations.Operator.Name.$t = coreData.operatorName;
+            resourceFrameToUpdate.organisations.Operator.ShortName.$t = coreData.operatorName;
```

Comment:
I think the purpose of these two attributes in the Netex is slightly separate. I'm not sure exactly what they should be, but I don't think they should both be `coreData.operatorName`. Looking at the old generators, I also think they both do things slightly differently.

Reply:
Yeah, my bad. Name should be from operator data.

Reply:
Looking at it, I'm pretty certain they're the same thing anyway. We do this check:

`const baseOperatorInfo = isSchemeOperatorTicket(userPeriodTicket)
        ? getBaseSchemeOperatorInfo(userPeriodTicket)
        : operatorData.find(operator => operator.operatorPublicName === userPeriodTicket.operatorName);`

to find which of the array of Operators is the base one, and I don't think we'd do that if they weren't the same anyway.

Reply:
You're right, it's a bit weird the way that works in the periodTicketNetexGenerator. Looking at the pointToPointTicketNetexGenerator, it doesn't have this check. I'll be honest, I'm not sure why theres a difference between the two.